src/Makefile.am

```diff
@@ -156,25 +156,20 @@ eos_updater_SOURCES = \
 
 dist_man8_MANS += docs/eos-updater.8
 
+eos_updater_dbus_if := com.endlessm.Updater
```

Comment:
Probably doesn’t make sense to have this as a variable. Changing the interface name would be a major change to all of the codebase, so factoring it out adds no value, and makes things less greppable (for example, `git grep com.endlessm.Updater.service` doesn’t work).